docs/articles/workflow/git-workflow.rst

```diff
@@ -24,6 +24,22 @@ developers, as it is the repository the developer will typically push to.  On th
 recommend to always type the remote and branch name when pushing, which forces the developer to be
 more explicit about their intentions.
 
+If you didn't manage to clone the repo via the steps above, it most likely due to the SSH key hasn't
+been set up yet. To quick fix ::
+
+    $ ssh-keygen -t rsa -b 4096 -C "your_email@example.com"
+    # This will generate a new ssh key for you
+
+    $ sudo apt-get install xclip
+    # Downloads and installs xclip. If you don't have `apt-get`, you might need to use another installer (like `yum`)
+
+    $ xclip -sel clip < ~/.ssh/id_rsa.pub
+    # Copies the contents of the id_rsa.pub file to your clipboard
+
+Click your avatar at GitHub page and go to Settings -> SSH and GPG keys, paste your key there.
+A long guide could be found here: https://help.github.com/articles/connecting-to-github-with-ssh/
```

Comment:
This is too github specific, can we make it more abstract by saying something like "then add it to the account on the git web frontend" or something like that?

Reply:
Sure, sounds good.